packs/equipment-effects/effect-lions-bite.json

```diff
@@ -0,0 +1,60 @@
+{
+    "_id": "cP9oWbEa7cJnnK9j",
+    "img": "icons/equipment/shield/kite-steel-boss-gold.webp",
+    "name": "Effect: Lion's Bite",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Lion's Shield]</p>\n<p>The shield's biting maw is a martial melee weapon that deals 2d6 piercing damage and has the deadly d6 trait; it can't be enhanced by runes.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder GM Core"
+        },
+        "rules": [
+            {
+                "category": "martial",
+                "damage": {
+                    "base": {
+                        "damageType": "piercing",
+                        "dice": 2,
+                        "die": "d6"
+                    }
+                },
+                "group": "shield",
+                "key": "Strike",
+                "label": "PF2E.SpecificRule.Equipment.LionsShield.LionsBiteLabel",
+                "range": null,
+                "traits": [
+                    "deadly-d6"
+                ]
+            },
+            {
+                "key": "WeaponPotency",
+                "label": "PF2E.PotencyRuneLabel",
+                "selector": "{item|id}-attack",
+                "value": 1
+            }
```

Comment:
It's not clear whether it should benefit from the potency rune here

Reply:
I think you're right about it not having the potency rune - fixed.

As for the WeaponPotency label, if it's not specified it defaults to the item name.
<img width="296" alt="Screen Shot 2024-06-21 at 10 59 20 AM" src="https://github.com/foundryvtt/pf2e/assets/1904698/76fca31f-cdcf-4394-9470-7b2488fdf7b3">